airbyte-integrations/connectors/source-google-analytics-data-api/source_google_analytics_data_api/authenticator.py

```diff
@@ -0,0 +1,75 @@
+import datetime
+
+import jwt
+import requests
+
+from source_google_analytics_data_api import utils
+
+
+class GoogleServiceKeyAuthenticator(requests.auth.AuthBase):
```

Comment:
is this completely unique such that we can't use any of the cdk authenticator frameworks?

Reply:
I didn't find any proper base class in `cdk` for service authentication. In some other connectors they are using OAuth class as a base class and re-implementing methods but it is basically the same as I do using `requests.auth.AuthBase`. It is more clea[n/r] I think